src/main/java/me/akainth/ambient/ui/SyncedTreeView.java

```diff
@@ -52,6 +52,7 @@ public SyncedTreeView(String label, @Nullable String defaultText, DocumentInterp
         clearTree();
         updatePreview();
 
+        tree.setRootVisible(false);
```

Comment:
Thank you for your contribution! The `SyncedTreeView` is also used by `SnarfModuleWizardStep`, and this would adversely affect it. It would be better to insert this in `SubmissionConfirmationDialog#createUIComponents`.

Reply:
Done. Although this does require reaching into the `SyncedTreeView` and modifying its `tree` (via `getTree()`). I think that's reasonable, but if you'd prefer I'll add a wrapping call (`SyncedTreeView.setRootVisible()`)